content/guides/component-testing/framework-specific-guides.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Framework Specific Guides
```

Comment:
Thinking out loud here of a better title for this page. If we're keeping it scoped to configuration then here are some options:

1. Framework Configuration
2. Framework Setup

Reply:
I put "Framework Configuration" for now, but I think we can probably think of something a little better. Let's keep thinking.

I added Next.js 4/5, I'll keep working through the other major frameworks (CRA, Nuxt).